tox.ini

```diff
@@ -143,11 +143,9 @@ commands =
     python {toxinidir}/scripts/pycodestyle_on_repo.py
     python {toxinidir}/scripts/run_pylint.py
 deps =
-    {[testing]deps}
+    {[testenv]deps}
     pycodestyle
     pylint >= 1.6.4
-setenv =
-    PYTHONPATH = {toxinidir}/_testing
```

Comment:
OK. `appveyor.yml` won't need it soon. The `datastore` emulator probably doesn't need it? Same with docs, since they use Python 2.7